src/transport/PairingSession.cpp

```diff
@@ -99,4 +100,22 @@ CHIP_ERROR PairingSession::DecodeMRPParametersIfPresent(TLV::Tag expectedTag, TL
     return tlvReader.ExitContainer(containerType);
 }
 
+void PairingSession::Clear()
+{
+    if (mSessionManager != nullptr)
+    {
+        auto handle = GetSecureSessionHandle();
```

Comment:
This doesn't look to be defined anywhere?

Reply:
Merge conflict...  Fixing.